README.md

```diff
@@ -1,32 +1,18 @@
-# HACBS Release bundles
+# Release Service Catalog
 
-Release service bundles is a curated Tekton bundles catalog that contains bundles that are used by the
-[Appstudio Release operator](https://github.com/redhat-appstudio/release-service) to process releases in the context of
+This repository contains a collection of Tekton resources to be used with the Release Service. The main purpose of these
+resources is to facilitate and streamline the release process.
 AppStudio.
 
-## Using the bundles
+# Resources
+Here's a brief overview of what you can find in the different directories of this catalog:
 
-All the bundles in this repository are pushed to our [Quay organization](https://quay.io/organization/hacbs-release).
-This push happens automatically whenever a new bundle is added to the repository or an existent one is updated. Two new
-tags will be created or updated during this automatic process:
+* `pipelines`: This directory contains Release Pipelines which use a collection of both, Release Tasks and Hub Tasks.
+* `tasks`: The tasks directory holds Tekton Tasks that are used in our Release Pipelines.
+* `hub`: This directory contains Tekton resources that have been sourced from the Tekton Hub.
 
-* **version**: This tag will point to the Catalog version of the Tekton resource, for example "0.1", 0.2", etc..
-* **main**: This tag will point always to the existing code in the repository main branch.
-* **<commit-sha>**: A tag with the commit-sha of the change will be created, so we can have multiple bundles per each
-change in the git repository.
+# Versioning
 
-## Adding new bundles
```

Comment:
Why are the Adding new bundles and Linting of yaml files sections removed? We still have linting. And adding new stuff has a new process, but it is still nice to say the process

Reply:
See comment below.